functions/common/csv-writer.js

```diff
@@ -0,0 +1,47 @@
+const fs = require('fs');
+const path = require('path');
+const os = require('os');
+
+class CsvWriter {
+  constructor (projName, rawData, desiredLanguage) {
+    this.projName_ = projName;
+    this.rawData_ = rawData;
+    this.desiredLanguage_ = desiredLanguage;
+    this.numberOfRecords_ = numberOfRecords;
+  }
+
+  getTmpCsvFile() {
```

Comment:
What's the purpose of writing a tmp file here? Can't we just write directly to the output stream?